admin/load.php

```diff
@@ -503,3 +508,129 @@ function perflab_dismiss_wp_pointer_wrapper() {
 	check_ajax_referer( 'dismiss_pointer' );
 }
 add_action( 'wp_ajax_dismiss-wp-pointer', 'perflab_dismiss_wp_pointer_wrapper', 0 );
+
+/**
+ * Callback function to handle admin scripts.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_modules_page_scripts() {
+	wp_enqueue_script( 'updates' );
+
+	wp_localize_script(
+		'updates',
+		'_wpUpdatesItemCounts',
+		array(
+			'settings' => array(
+				'totals' => wp_get_update_data(),
+			),
+		)
+	);
+
+	wp_enqueue_script( 'thickbox' );
+	wp_enqueue_style( 'thickbox' );
+}
+
+/**
+ * Callback function hooked to admin_action_perflab_activate_plugin to handle plugin activation.
+ *
+ * @since n.e.x.t
+ */
+function perflab_activate_plugin() {
+	// Do not proceed if plugin query arg is not present.
+	if ( empty( $_GET['plugin'] ) ) {
+		return;
+	}
+
+	// The plugin being activated.
+	$plugin = sanitize_text_field( $_GET['plugin'] );
```

Comment:
Annoyingly, for back-compat, WordPress [adds](https://github.com/WordPress/WordPress/blob/27720e15a79548313a5f647f373494d13263379c/wp-includes/load.php#L915-L940) "magic" quotes to all input variables even if this obsolete feature of PHP is turned off. So to ensure that no backslashes appear in values obtained from input vars, you should always unslash. I'm surprised that `WordPress.Security.ValidatedSanitizedInput.MissingUnslash` isn't getting raised by PHPCS here (maybe due to https://github.com/Automattic/VIP-Coding-Standards/issues/425?).

```suggestion
	$plugin = sanitize_text_field( wp_unslash( $_GET['plugin'] ) );
```